web/src/components/vis/ScreePlotTile.vue

```diff
@@ -1,42 +1,42 @@
-<script>
+<script lang="ts">
+import { defineComponent, computed, ref } from '@vue/composition-api';
 import ScreePlot from '@/components/vis/ScreePlot.vue';
 import VisTile from '@/components/vis/VisTile.vue';
-import plotData from './mixins/plotData';
-
-export default {
-  components: {
-    ScreePlot,
-    VisTile,
-  },
-
-  mixins: [plotData('pca')],
+import usePlotData from './use/usePlotData';
```

Comment:
Is this `use` directory a recommended best practice for storing mixins?

Reply:
I'm copying Brandon's `useAxisPlot`, I don't know if it's a best practice.